src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -4,7 +4,8 @@
 import logging
 import cherrypy
 
-from . import ApiController, RESTController
+from . import ApiController, RESTController, \
+    allow_empty_body
```

Comment:
Just a hint: if you don't want to explicitly have to import a specific helper method/class, you can use a related class as a 'namespace' container, so perhaps this could hang from `RESTController` (same as `@RESTController.Resource`?:
```py
from . import RESTController

@RESTController.allow_empty_body
def ...
```

Reply:
Yes it can be done that way, but few controllers (for e.g. rgw buckets/users) use ApiController class too, which uses Endpoint method of it (ApiController class)